cli/src/movement_util.rs

```diff
@@ -214,16 +214,14 @@ fn choose_commit<'a>(
     choices.push("q".to_string());
     drop(formatter);
 
-    let choice = ui.prompt_choice(
+    let index = ui.prompt_choice(
         "enter the index of the commit you want to target",
         &choices,
         None,
     )?;
-    if choice == "q" {
```

Comment:
I don't think the line above is valid anymore?
```rs
    writeln!(formatter, "q: quit the prompt")?;
```

Perhaps remove or change `q` to `^C`?

Reply:
No, `q` still works because `commits[q_index]` doesn't exist. I personally don't think `q` is needed, though. `^D` should always work.

Reply:
I agree with that, no need to add special handling when the terminal behaviour works